moped-editor/src/utils/dateAndTime.js

```diff
@@ -18,6 +18,8 @@ export const formatDateType = (dateString) => {
  * @return {String} Date formatted as MM/DD/YYYY
  */
 export const formatTimeStampTZType = (timeStampTZString) => {
+  if (timeStampTZString === null) return "";
```

Comment:
this helper was returning bad dates when on phase dates that were `null` in the DB